vNext/shared/AppInsightsCore/src/JavaScriptSDK/AppInsightsCore.ts

```diff
@@ -208,7 +208,7 @@ export class AppInsightsCore implements IAppInsightsCore {
     /**
      * Periodically check logger.queue for
      */
-    pollInternalLogs(): number {
+    pollInternalLogs(intervalMS?: number, eventName?: string): number {
```

Comment:
Why adding intervalMS?, I believe config.diagnosticLogInterval is used for this purpose already

Reply:
removed